qutip/tests/test_wigner.py

```diff
@@ -36,11 +36,158 @@
 from numpy.random import rand
 from numpy.testing import assert_, run_module_suite, assert_equal
 
-from qutip.states import coherent, fock
-from qutip.wigner import wigner
+from qutip.states import coherent, fock, ket, bell_state
+from qutip.wigner import wigner, wigner_transformation, _parity
 from qutip.random_objects import rand_dm, rand_ket
 
 
+def test_wigner_bell1_su2parity():
+    psi = bell_state('00')
+
+    steps = 100
+    theta = np.tile(np.linspace(0, np.pi, steps), 2).reshape(2, steps)
+    phi = np.tile(np.linspace(0, 2 * np.pi, steps), 2).reshape(2, steps)
+    slicearray = ['l', 'l']
+
+    wigner_analyt = np.zeros((steps, steps))
+    for t in range(steps):
+        for p in range(steps):
+            wigner_analyt[t, p] = ((1 + np.sqrt(3) * np.cos(theta[0, t]))
+                                   * (1 + np.sqrt(3) * np.cos(theta[1, t]))
+                                   + 3 * (np.sin(theta[0, t])
+                                   * np.exp(-1j * phi[0, p])
+                                   * np.sin(theta[1, t])
+                                   * np.exp(-1j * phi[1, p])
+                                   + np.sin(theta[0, t])
+                                   * np.exp(1j * phi[0, p])
+                                   * np.sin(theta[1, t])
+                                   * np.exp(1j * phi[1, p]))
+                                   + (1 - np.sqrt(3) * np.cos(theta[0, t]))
+                                   * (1 - np.sqrt(3)
+                                   * np.cos(theta[1, t]))) / 8.
```

Comment:
The formula return a complex but `wigner_analyt` is real resulting in a warning.
Please add a `np.real` to remove the warning.